server/utils/matchUtils.test.ts

```diff
@@ -256,43 +270,47 @@ describe('matchUtils', () => {
     })
   })
 
-  describe('summaryCardHeader', () => {
+  describe('summaryCardLink', () => {
     it('returns a link to the confirm page with the premises name and bed', () => {
-      const bedSearchResult = bedSearchResultFactory.build()
+      const spaceSearchResult = spaceSearchResultFactory.build()
       const placementRequestId = '123'
       const startDate = '2022-01-01'
       const durationWeeks = '4'
       const durationDays = '1'
+      const durationInDays = Number(durationWeeks) * 7 + Number(durationDays)
 
-      summaryCardHeader({ bedSearchResult, placementRequestId, startDate, durationDays, durationWeeks })
+      summaryCardLink({
+        spaceSearchResult,
+        placementRequestId,
+        startDate,
+        durationDays,
+        durationWeeks,
+      })
 
-      expect(linkTo).toHaveBeenCalledWith(
-        paths.placementRequests.bookings.confirm,
-        { id: placementRequestId },
-        {
-          text: `${bedSearchResult.premises.name} (Bed ${bedSearchResult.bed.name})`,
-          query: {
-            bedSearchResult: encodeBedSearchResult(bedSearchResult),
+      expect(
+        `${paths.placementRequests.bookings.confirm({ id: placementRequestId })}${createQueryString(
+          {
+            spaceSearchResult: encodeSpaceSearchResult(spaceSearchResult),
```

Comment:
again, here I'm not expecting any encoding, I'm hoping to pull startDate, duration, premisesName, premisesId, apType from req.query.